boltdesignsystem/bolt
# Fix: update xxsmall font-size to follow conventions

## Jira

http://vjira2:8080/browse/BDS-2054

## Summary

Fixes the xxsmall font-size so it consistent with how other font-sizes were created.

## Details

1. Added xxsmall variables
2. Updated font-size map to be consistent

## How to test

Run the branch locally and create a dummy page, test out all sizes of the Headline component, make sure xxsmall is the smallest of them all.
